components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -1688,6 +1691,21 @@ subroutine ocn_diagnostic_solve_vortVel(relativeVorticity, &
       !$omp end parallel
 #endif
 
+      !LVR: 20201204 -- add in new vertical velocity prognostic variable
+!      if(.not. config_enable_nonhydrostatic_mode .or. config_enable_vertMomentum_disable_nonHydro) then
+!      !if (config_enable_nonhydrostatic_mode) then
+!        !$omp parallel
+!        !$omp do schedule(runtime) &
+!        !$omp private(k)
+!        do iCell=1,nCells
+!          do k=1,maxLevelCell(iCell)+1
+!            vertVelocityNonhydro(k,iCell) = vertVelocityTop(k,iCell)
+!          enddo
+!        enddo
+!        !$omp end do
+!        !$omp end parallel
+!      endif
+
```

Comment:
It's weird to merge in a block of commented out code.  Should we just remove this?  I was never confident this was a correct operation.

Reply:
Yes, please remove this.  I agree that we have a general policy against committing commented-out code.